src/Foundation.hs

```diff
@@ -106,6 +106,7 @@ instance Yesod App where
 
         manalytics <- appAnalytics . appSettings <$> getYesod
         isSearch <- testCurrentRoute (== SearchR)
+        searchText <- map (maybe "" unpack) (lookupGetParam "q")
```

Comment:
Would `fromMaybe ""` work here? I think the templates ought to work the same way when splicing `Text` values in as opposed to `String` values.